OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameMenuLogic.cs

```diff
@@ -351,12 +351,26 @@ void CreateSaveMapButton()
 				hideMenu = true;
 				var editorActorLayer = world.WorldActor.Trait<EditorActorLayer>();
 				var actionManager = world.WorldActor.Trait<EditorActionManager>();
+
+				var playerDefinitions = editorActorLayer.Players.ToMiniYaml();
+
+				var playerCount = new MapPlayers(playerDefinitions).Players.Count;
+				if (playerCount > MapPlayers.MaximumPlayerCount)
+				{
+					ConfirmationDialogs.ButtonPrompt(
+					title: "Error: Max player count exceeded",
```

Comment:
Wrong indentation on this and following lines?

Reply:
Good catch, fixed.